### DISCUSSION TITLE
The road to ActivityPods 2.0

### DISCUSSION THREAD

**Discussion by giscus** [assemblee-virtuelle/activitypods#148] [Blog articles]
# The road to ActivityPods 2.0

Since this summer, thanks to a grant from NLnet as part of the NGI0 Entrust Fund program, we've been working hard on ActivityPods 2.0, with a planned release in spring 2024. The aim of this article is to explain this background work and what's at stake.

https://activitypods.org/the-road-to-activitypods-2-0/

**Comment by srosset81** [comment]
Feel free to post comments about this article here :-)
